src/utils/source.js

```diff
@@ -484,3 +484,7 @@ export function getSourceQueryString(source: ?Source) {
 
   return parseURL(getRawSourceURL(source.url)).search;
 }
+
+export function isUrlExtension(url: string) {
+  return /^(chrome|moz)-extension:\/\//.test(url);
```

Comment:
@konradk Just a heads up: I also have at least one extension (namely [User Agent Switcher](https://addons.mozilla.org/en-US/firefox/addon/uaswitcher/)) whose scheme seems to end by only *one* trailing slash and not two. The above regex tests with two trailing slashes at the end.

See screenshot:

![image](https://user-images.githubusercontent.com/12525563/51108747-6d509480-17f3-11e9-8660-4bd96f973eab.png)

Pinging also @AnshulMalik because I think this is probably relevant for https://bugzilla.mozilla.org/show_bug.cgi?id=1518221 as well.

Note sure how to handle this (and why this is under `blob://`) but testing against 
```js
// one trailing slash
return /^(chrome|moz)-extension:\//.test(url);
```
and not 
```js
// two trailing slashes
return /^(chrome|moz)-extension:\/\//.test(url);
```
would be worthwhile.

If so, probably also good if a test would be added for this edge case.

Reply:
@gerdneuman Nice catch, I will fix that.

Reply:
Not sure why it's blob and has just one `/`